src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,45 @@
 package mate.academy;
 
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private CopyOnWriteArrayList<EventListener> events = new CopyOnWriteArrayList<>();
```

Comment:
The 'events' field is being initialized twice: once here and again in the constructor on line 13. It's unnecessary to initialize it here since the constructor will always be called when creating an instance of EventManager.